syft/workers/socketio_client.py

```diff
@@ -0,0 +1,84 @@
+import time
+from typing import List
+from typing import Union
+
+import socketio
+import torch
+
+import syft as sy
+from syft.frameworks.torch.tensors.interpreters import AbstractTensor
+from syft.workers.virtual import VirtualWorker
+
+
+class WebsocketIOClientWorker(VirtualWorker):
+    """A client which will send a message to a WebsocketIOServerWorker when it receives
```

Comment:
Change this to google docstring style. Example: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html